corehq/apps/registration/views.py

```diff
@@ -60,6 +60,9 @@
 ], send_to_ops=False, fail_if_debug=False)
 
 
+RESEND_INVITATION_TIME_INTERVAL = 60  # seconds
```

Comment:
This isn't resending an invitation. How about `CONFIRMATION_RESEND_LIMIT_SECONDS`?

Reply:
No feedback from my side. Change makes sense. Thanks.